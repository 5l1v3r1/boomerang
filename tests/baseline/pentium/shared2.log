Unknown library function proc1
assgn   42 *32* %pc := m[0x8049c18]
undoComputedBB for statement   43 <all> := CALL __thunk_36_foo__1D(<all>)
              Reaching definitions: r24=r24{20},   r25=r25{18},   r26=r26{18},   r27=r27{21},   r28=r28{40},
                r29=r29{3},   r30=r30{19},   tmp1=tmp1{10},   %flags=SUBFLAGS32( tmp1{10}, 24, r28{11} ),   %ZF=%ZF{12}
                %CF=%CF{12}
              Live variables: r28,  r30
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 44 to local table
in proc main adding addrExp r28{0} - 48 to local table
in proc main adding addrExp r28{0} - 44 to local table
in proc main adding addrExp r28{0} - 48 to local table
Unknown library function proc2
Unknown library function proc3
Unknown library function proc4
Unknown library function proc5
after removing matching assigns (%pc).
in proc __thunk_36_foo__1D adding addrExp r28{0} - 4 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 8 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 12 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 24 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 28 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 32 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 28 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 32 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 24 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 28 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 32 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 24 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 28 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 32 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 28 to local table
in proc __thunk_36_foo__1D adding addrExp r28{0} - 32 to local table
finding final parameters for __thunk_36_foo__1D
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
could not set type for unknown parameter expression m[r28{0} + 4]
could not set type for unknown parameter expression m[r28{0} + 8]
could not set type for unknown parameter expression m[r28{0} + 12]
finding final parameters for __thunk_36_foo__1D
local esp has type unsigned int that doesn't agree with type of define void of a library, why?
