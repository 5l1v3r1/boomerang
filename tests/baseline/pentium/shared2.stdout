Boomerang alpha 0.3.1 09/Sep/2006
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 1 procs
decompiling...
 considering main
 decompiling main
 considering __thunk_36_foo__1D
 decompiling __thunk_36_foo__1D
global type analysis for main
global type analysis for __thunk_36_foo__1D
generating code...
output written to ./tests/outputs/pentium/shared2
completed in 0 secs.
